VEDA_OS01/transcripts.py

```diff
@@ -38,11 +40,19 @@
 CIELO24_TRANSCRIPT_COMPLETED = django.dispatch.Signal(providing_args=['job_id', 'lang_code', 'org', 'video_id'])
 CONFIG = utils.get_config()
 
+
+# Cielo24 API version
+CIELO24_API_VERSION = 1
```

Comment:
currently, we do not use it in cielo24 callback/client, should this PR also cover that?